.circleci/config.yml

```diff
@@ -32,18 +32,26 @@
             POSTGRES_PASSWORD: ""
         - image: circleci/redis:3.2-alpine # 3.2.4 engine on AWS Elasticache
 
+  defaults: &defaults
+    working_directory: ~/vets-api
+    executor: vets_api_docker_executor
+
   jobs:
     verify_pacts:
-      working_directory: ~/vets-api
+      <<: *defaults
       executor: vets_api_docker_executor
```

Comment:
the defaults would also be pulling in the `executor` too, right?

Reply:
Yes, good catch! Forgot to remove